test/narjure/test/concept_creator.clj

```diff
@@ -0,0 +1,24 @@
+(ns narjure.test.concept-creator
```

Comment:
I would like to see more tests here, currently it looks like test of clojure's hash-map and atom